test/gtest/uct/ib/test_rc.cc

```diff
@@ -193,15 +199,16 @@ UCS_TEST_P(test_rc_iface_address, size_no_flush_remote)
     map_size_t sizes = {
         {"rc_mlx5", {7, 1}},
         {"dc_mlx5", {0, 5}},
-        {"rc_verbs", {7, 0}},
+        {"rc_verbs", {4, 0}},
     };
     check_sizes(m_entity, sizes);
 }
 
 UCS_TEST_P(test_rc_iface_address, size_flush_remote)
 {
+    int flush_rkey_enabled = rc_iface_flush_rkey_enabled(m_entity_flush_rkey);
     map_size_t sizes = {
-        {"rc_mlx5", {10, 1}},
+        {"rc_mlx5", {flush_rkey_enabled ? 10 : 7, 1}},
```

Comment:
why need to check here? Should not it be always enabled?

Reply:
No. Think about the scenario when using [&UCT_IB_MD_OPS_NAME(dv)](https://github.com/openucx/ucx/blob/v1.15.0-rc1/src/uct/ib/base/ib_md.c#L231).

Reply:
can you pls elaborate? Why wasn't the test failing before?

Reply:
@brminich configure with "--with-devx=no" option and run the test again.

Reply:
ok, shall we add this config to our gtest then?

Reply:
Maybe not.
Adding it into CI test cases will make the CI pipeline too long(e.g. 1. only ```verbs```, 2. only ```verbs``` & ```dv```, 3. all ```verbs``` & ```dv``` & ```devx```)
This problem is found when I'm running some test cases to help me understand the code with using ```dv```.